spec/lucky_cli/runner_spec.cr

```diff
@@ -4,11 +4,13 @@ include HaveDefaultHelperMessageMatcher
 
 describe LuckyCli::Runner do
   it "adds tasks to the runner when task classes are created" do
-    LuckyCli::Runner.tasks.map(&.name).reject(&.==("lucky_cli.dev")).should eq [
-      "another_task",
-      "my.cool_task",
-      "my.custom_name",
-    ]
+    expected_task_names = ["another_task", "my.cool.task", "my.custom_name"]
```

Comment:
This is why the tests are failing
```suggestion
    expected_task_names = ["another_task", "my.cool_task", "my.custom_name"]
```

Reply:
ha! Good catch 😅  For some reason all of the specs fail locally on my machine, so not sure which ones are "real" errors and which are just my machine being weird 😕  I should take some time to figure that out.